nautobot/dcim/utils.py

```diff
@@ -38,6 +36,9 @@ def cable_status_color_css(record):
     """
     Given a record such as an Interface, return the CSS needed to apply appropriate coloring to it.
     """
+    # Avoid circular import
+    from nautobot.core.utils import hex_to_rgb, lighten_color, rgb_to_hex
```

Comment:
Not a fan of this - it should be safe to import anything from core. This feels like a step backwards.